services/web/src/screens/Organizations/List/index.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Table, Divider, Button, Message } from 'semantic';
+import { formatDateTime } from 'utils/date';
+import { request } from 'utils/api';
+import { screen } from 'helpers';
+import {
+  Confirm,
+  HelpTip,
+  Breadcrumbs,
+  SearchProvider,
+  Layout,
+} from 'components';
+
+import Filters from 'modals/Filters';
+import EditOrganization from 'modals/EditOrganization';
+
+@screen
+export default class OrganizationList extends React.Component {
+  onDataNeeded = async (params) => {
+    const { category, ...rest } = params;
+    return await request({
+      method: 'POST',
+      path: '/1/organizations/search',
+      body: {
+        ...rest,
+        ...category && { categories: [category.id] },
+      },
+    });
+  };
+
+  fetchCategories = async (query) => {
+    const { data } = await request({
```

Comment:
what about meta ? shouldnt that also be pass on the search provider in case of pagination

Reply:
this one was added by the generator actually wasn't even used, just a mistake